kube-aliases.plugin.zsh

```diff
@@ -0,0 +1,8 @@
+KALIAS=$ZSH_CUSTOM/plugins/kube-aliases
+KRESOURCES=$ZSH_CUSTOM/plugins/kube-aliases/docs/resources
+SHELL_NAME=$(basename $SHELL)
+
+# Auto complete, for bash replace zsh with bash
+# For some reason this is sourcing the oh-my-zsh plugin.
+source <(kubectl completion $SHELL_NAME)
```

Comment:
Im not sure we need to do this for people. Im not sure I would think this is something we would be doing as a user. Id probably already have added autocomplete. Ill leave this up to you though.